cmd/dep/status.go

```diff
@@ -502,7 +616,7 @@ outer:
 	}
 	out.MissingFooter()
 
-	return digestMismatch, hasMissingPkgs, nil
+	return digestMismatch, hasMissingPkgs, 0, nil
```

Comment:
This one could be `true, hasMissingPkgs...`

And then line 566 can go away too.